docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -47,6 +47,20 @@ Incidents are raised as they would in any Camunda cluster. Use the variables and
 
 To start a new process instance, click on the **Restart process** button on the start event, or return to the definition view by clicking **View all** on the top banner.
 
+## Rewind a process
+
+After completing part of your process, you can **rewind** to a previous task to test a section of your process. Play will start a new instance and replay your actions up to the selected previous task.
```

Comment:
Here you say `task`, but elsewhere you say `element`, which may include events.

Reply:
I copied this explanation from old Play's docs to be honest 😅  it seems element is more appropriate then, I'll fix it

Reply:
Got it 😁 
I didn't pay as much attention to those docs